pep-0539.txt

```diff
@@ -209,10 +210,11 @@ Platform Support Changes
 A new "Native Thread Implementation" section will be added to PEP 11 that
 states:
 
-* As of CPython 3.7, in the case of enabling thread support, all platforms are
-  required to provide at least one of native thread implementation (as of
-  pthreads or Windows) to implement TSS API. Any TSS API problems that occur
-  in the implementation without native thread will be closed as "won't fix".
+* As of CPython 3.7, in the case of enabling threads support in the
+  interpter, all platforms are required to provide at least one of native
```

Comment:
The note `in the case of enabling threads support in the  interpter,` no longer makes any sense, it could be removed.